tests/components/zone/test_init.py

```diff
@@ -1,229 +1,224 @@
-
-    def test_active_zone_skips_passive_zones(self):
-        """Test active and passive zones."""
-        assert setup.setup_component(
-            self.hass,
-            zone.DOMAIN,
-            {
-                "zone": [
-                    {
-                        "name": "Passive Zone",
-                        "latitude": 32.880600,
-                        "longitude": -117.237561,
-                        "radius": 250,
-                        "passive": True,
-                    }
-                ]
-            },
-        )
-        self.hass.block_till_done()
-        active = zone.async_active_zone(self.hass, 32.880600, -117.237561)
-        assert active is None
-
-    def test_active_zone_skips_passive_zones_2(self):
-        """Test active and passive zones."""
-        assert setup.setup_component(
-            self.hass,
-            zone.DOMAIN,
-            {
-                "zone": [
-                    {
-                        "name": "Active Zone",
-                        "latitude": 32.880800,
-                        "longitude": -117.237561,
-                        "radius": 500,
-                    }
-                ]
-            },
-        )
-        self.hass.block_till_done()
-        active = zone.async_active_zone(self.hass, 32.880700, -117.237561)
-        assert "zone.active_zone" == active.entity_id
-
-    def test_active_zone_prefers_smaller_zone_if_same_distance(self):
-        """Test zone size preferences."""
-        latitude = 32.880600
-        longitude = -117.237561
-        assert setup.setup_component(
-            self.hass,
-            zone.DOMAIN,
-            {
-                "zone": [
-                    {
-                        "name": "Small Zone",
-                        "latitude": latitude,
-                        "longitude": longitude,
-                        "radius": 250,
-                    },
-                    {
-                        "name": "Big Zone",
-                        "latitude": latitude,
-                        "longitude": longitude,
-                        "radius": 500,
-                    },
-                ]
-            },
-        )
+    assert await setup.async_setup_component(hass, zone.DOMAIN, {"zone": info})
+
+    assert len(hass.states.async_entity_ids("zone")) == 2
+    state = hass.states.get("zone.test_zone")
+    assert info["name"] == state.name
+    assert info["latitude"] == state.attributes["latitude"]
+    assert info["longitude"] == state.attributes["longitude"]
+    assert info["radius"] == state.attributes["radius"]
+    assert info["passive"] == state.attributes["passive"]
+
+
+async def test_setup_zone_skips_home_zone(hass):
+    """Test that zone named Home should override hass home zone."""
+    info = {"name": "Home", "latitude": 1.1, "longitude": -2.2}
+    assert await setup.async_setup_component(hass, zone.DOMAIN, {"zone": info})
+
+    assert len(hass.states.async_entity_ids("zone")) == 1
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4870504)